grunt/helpers.js

```diff
@@ -195,6 +212,23 @@ module.exports = function(grunt) {
         }
     };
 
+    exports.isPluginScriptSafe = function(pluginPath) {
+
+        pluginPath = pluginPath.replace(convertSlashes, "/");
+        var includes = grunt.config('scriptSafe');
+        var isExplicitlyDefined = (includes && pluginPath.search(exports.getScriptSafeRegExp()) !== -1);
+        var isIncluded = grunt.option('allowscripts') || includes[0] === "*" || isExplicitlyDefined;
+
+        if (!isIncluded) {
+            //grunt.log.writeln('Excluded ' + chalk.red(pluginPath));
+        } else {
+            //grunt.log.writeln('Included ' + chalk.green(pluginPath));
+        }
```

Comment:
Should this `if` statement be removed now that the log statements have been commented out?